### DISCUSSION TITLE
How do I replace jsdelivr with unpkg

### DISCUSSION THREAD

**Discussion by thesixonenine** [CaiJimmy/hugo-theme-stack#910] [Q&A] [answered]
How do I replace jsdelivr with unpkg?

**Comment by quivenda** [comment, marked as answer]
Copy `themes\hugo-theme-stack\data\external.yaml` file to the Hugo root directory `data\external.yaml` so that when you update the theme, it won't overwrite the old one.

Thereafter, just replace `jsdeliver` link with `unpkg`.

**Reply by thesixonenine** [reply]
thanks.
